site/content/docs/11-deploying.md

```diff
@@ -2,7 +2,17 @@
 title: Deployment
 ---
 
-Sapper apps run anywhere that supports Node 8 or higher.
+Sapper apps run anywhere that supports Node 10 or higher.
+
+### Deploying from sapper build
+
+If you're deploying your app to a different machine than the one you're building on, you will need to copy your `package.json` over and install your production dependencies (required by `server.js`) using `--only=prod` (or `--only=production`):
+
+```bash
+npm install --only=prod
+```
+
+Other than `__sapper__/build` directory and `package.json`, you will also need to include your `static` directory and its contents.
```

Comment:
I don't think you actually need the `package.json` to be deployed to the prod machine

```suggestion
You will also need to run `npm run build` and then make the `__sapper__/build` and `static` directories available on the production machine (either by building it there or by copying over the resulting files).
```

Reply:
I agree, it's the same case as the first statement where it can be both copied or built directly. I think we should also say how to run the app (like building section) to replace `npm start` that `package.json` included, something like

> You can then start your app by running `node __sapper__/build`

Aside from that, perhaps it's better to just make sure `__sapper__/build` is included at the start of this section since `npm run build` has been mentioned in building section.

Reply:
> You can then start your app by running `node __sapper__/build`

That makes sense to me

> Aside from that, perhaps it's better to just make sure __sapper__/build is included at the start of this section since npm run build has been mentioned in building section.

I think it's okay to mention that `npm run build` should be run. I'd rather be more explicit than leave out a step